server/src/test/scala/org/http4s/server/middleware/TimeoutSpec.scala

```diff
@@ -45,22 +46,20 @@ class TimeoutSpec extends Http4sSpec {
       checkStatus(altTimeoutService.orNotFound(neverReq), customTimeout.status)
     }
 
-    "clean up resources of the loser" in {
-      val clean = new AtomicBoolean(false)
+    "cancel the loser" in {
+      val cancelled = new AtomicBoolean(false)
```

Comment:
English uses both, but cats-effect uses "canceled".

It's not part of the API, so I won't die on this hill.

Reply:
Nitpicky much...